host.go

```diff
@@ -206,9 +206,27 @@ func (h *Host) ConfigureAuth() error {
 		return nil
 	}
 
-	ip, err := h.Driver.GetIP()
-	if err != nil {
-		return err
+	var (
+		ip         = ""
+		ipErr      error
+		maxRetries = 4
+	)
+
+	for i := 0; i < maxRetries; i++ {
+		ip, ipErr = h.Driver.GetIP()
```

Comment:
Consider `log.Debugf`ing this `ipErr` since if something funky happens we want to know about it.